ui/admin/app/templates/scopes/scope/credential-stores/credential-store/worker-filter.hbs

```diff
@@ -36,6 +36,7 @@
         @language='bash'
         @value={{@model.worker_filter}}
         @hasCopyButton={{true}}
+        @isOptional={{true}}
```

Comment:
This needs to be added to `ui/admin/app/components/form/credential-store/vault/index.hbs`

Reply:
Lol yea I saw that after I pushed so I pushed again with the same commit text